components/hab/src/cli/hab.rs

```diff
@@ -163,6 +166,9 @@ pub enum Channel {
 #[derive(ConfigOpt, StructOpt)]
 #[structopt(no_version, aliases = &["j", "jo"], settings = &[AppSettings::ArgRequiredElseHelp, AppSettings::SubcommandRequiredElseHelp])]
 /// Commands relating to Habitat Builder jobs
```

Comment:
I am just thinking whether we should add `(Deprecated and UNSUPPORTED)` or something similar at the end - so this is readily available in the help when someone runs the command `hab bldr help`

Reply:
Went with prefixing 'REMOVED: '